src/wincng.h

```diff
@@ -299,21 +302,20 @@ struct _libssh2_wincng_cipher_ctx {
 struct _libssh2_wincng_cipher_type {
     BCRYPT_ALG_HANDLE *phAlg;
     unsigned long dwKeyLength;
-    unsigned long dwUseIV;
+    int useIV;
+    int ctrMode;
```

Comment:
I actually like using bool, but as we have no code already using it within libssh2 and it isn't a native type in C89 it needs to be used with care if we're going to introduce it now. So, the easy route is to add a comment about it but avoid it for now until we have someone introduce it properly!